dockerize.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+IMAGE_NAME="boresha/agriculture-core"
+DOCKER_HOME="/agiculture-core/"
+ARG_NUM=2
+
+if [ "$1" == "init" ]; then
+	docker build -t $IMAGE_NAME $DIR
+elif [ "$1" == "build" ]; then
+	docker run --rm -v $(pwd):/agriculture-core/ $IMAGE_NAME npm build
+elif [ "$1" == "run" ]; then
+    docker run --rm --volume "$(pwd):/agriculture-core/" -ti agriculture-core ./start.sh
```

Comment:
I think maybe we don't need a start.sh script, we can just have 2 commands in this elif statement (one for yarn install and one for npm start). That way, people won't accidentally use the start script on it's own.. unless we want that?

Reply:
actually yea I think I agree with bea here, we'll just use two `docker run` commands.